db/migrate/20160106051255_recreate_people.rb

```diff
@@ -0,0 +1,54 @@
+class RecreatePeople < ActiveRecord::Migration
+  def change
+    # note, wasn't able to just drop and re-add table because of foreign key constraints
+    # (and apparently with postgres, there is no way to temporarily suspend the constraints)
+    # drop_table :people
+    # create_table :people do |t|
+    #   ## base Contact fields
+    #   t.references :division, index: true
+    #   t.string :name   # note agreeing with mel's change of this from 'display_name' as a more consistent field to hold the full name
```

Comment:
Great!  Sounds like we're on the same page.  I hadn't seen a well articulated discussion of this before. but I've always personally thought a single name field was a better approach than first/last fields, but that has generally been a lonely position for me, so I didn't even bother suggesting on this project.  
We'll likely still need to figure out the best way to handle a version sortable by last name, but we can deal with that at a later iteration.